lib/Widgets/TerminalView.vala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2025 elementary, Inc. (https://elementary.io)
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+[Version (since = "7.7.0")]
+public class Granite.TerminalView : Gtk.ScrolledWindow {
+
+    public TerminalView () {
+        this.add_css_class (Granite.CssClass.TERMINAL);
+    }
```

Comment:
This should be in a `construct` block. That's the block that runs for every constructor